### PR TITLE
Meetups: options if you can't find a speaker

### PR DESCRIPTION
**Pull request by mjang**
Applies to issue #72

### CODE REVIEW

**Comment by mjang** [comment]
I've asked @meker12 to join us, but I can't figure out how to add her to the reviewer list

**Comment by plaindocs** [comment]
This is failing due to "Write The Docs" instead of "Write the Docs"

**Comment by plaindocs** [comment]
@mjang I've added her to the team - once she accepts you should be able to add her as reviewer.

**Comment by meker12** [comment]
Looks like someone already added some info about remote speakers. The following paragraph appears after the list of alternatives to speakers:

If you can't find a local speaker, find a video of a talk from a previous Write the Docs conference <http://www.writethedocs.org/conf/>. Ask the speaker(s) if they're willing to field questions. Share the video with the group during the meetup. Set up questions and answers with the speaker during the Meetup. For one successful example, see the following Meetup ...


So a suggested revision: 

Remove the text you added above.

Move the paragraph about "if you can't find a local speaker..."  to the end of the "Speaker" subsection.

Suggested rewording:

If you can't find a local speaker, consider screening a popular talk from one of the
`Write the Docs conferences <http://www.writethedocs.org/videos/>`_  and inviting the conference presenter to call in for a live Q & A session.  Both the Boulder and Austin meetups had good luck with this format.
- `So you need to document an API <https://www.meetup.com/Boulder-Denver-WriteTheDocs-Meetup/events/232962552/>`_ remotely to provide live Q & A about the talk.
- `Living documentation<https://www.meetup.com/WriteTheDocs-ATX-Meetup/events/231859187/>`_

If you choose this format,  make sure your meeting venue supports high quality video presentation and phone conferencing.

----
Or you could add the remote speaker format at the end of the list of alternatives with the details following.

Actually, the other format is to just host a remote speaker with no conference talk like Kristof is doing for the London meetup: https://www.meetup.com/Write-The-Docs-London/events/236170664/


@mjang Thoughts?

File: docs/organizer-guide/meetups/starting.rst [hunk 1]
```diff
@@ -224,3 +224,16 @@ After Your Meetup
...
+---------------------------
+
+It is not always easy to find a speaker. Other meetups have done well with
+alternatives:
+
+* Coffee Klatches; suitable for meetups with a few people.
+* View a Write the Docs Conference Talk, available from our video library.
+Some meetups have combined this with live Q&A with the speaker
+* Pure networking meetup; you can "roundtable" a specific topic; give each
+attendee a chance to talk about what they do
```
Comment (meker12) [inline docs/organizer-guide/meetups/starting.rst]: Looks like someone already added some info about remote speakers. The following paragraph appears after the list of alternatives to speakers:

If you can't find a local speaker, find a video of a talk from a previous Write the Docs conference http://www.writethedocs.org/conf/. Ask the speaker(s) if they're willing to field questions. Share the video with the group during the meetup. Set up questions and answers with the speaker during the Meetup. For one successful example, see the following Meetup ...

So a suggested revision:

Remove the text you added above.

Move the paragraph about "if you can't find a local speaker..." to the end of the "Speaker" subsection.

Suggested rewording:

If you can't find a local speaker, consider screening a popular talk from one of the
Write the Docs conferences <http://www.writethedocs.org/videos/>_ and inviting the conference presenter to call in for a live Q & A session. Both the Boulder and Austin meetups had good luck with this format.

    So you need to document an API <https://www.meetup.com/Boulder-Denver-WriteTheDocs-Meetup/events/232962552/>_ remotely to provide live Q & A about the talk.
    Living documentation<https://www.meetup.com/WriteTheDocs-ATX-Meetup/events/231859187/>_

If you choose this format, make sure your meeting venue supports high quality video presentation and phone conferencing.

Or you could add the remote speaker format at the end of the list of alternatives with the details following.

Actually, the other format is to just host a remote speaker with no conference talk like Kristof is doing for the London meetup: https://www.meetup.com/Write-The-Docs-London/events/236170664/

**Comment by mjang** [comment]
@meker12 A git blame made me realize -- I was the one who added info about remote speakers... (and completely forgot to look when I did this PR.)

I should have time to revise this PR tomorrow afternoon.

**Comment by meker12** [comment]
lol

**Comment by plaindocs** [comment]
Totally! They should have a version of `git blame` that is `git wasitme?`

**Comment by mjang** [comment]
I screwed up a rebase, so I'm closing this and starting a new PR

**Pull request closed without merging**